mountinfo/mounted_linux_test.go

```diff
@@ -179,16 +179,21 @@ func requireOpenat2(t *testing.T) {
 	if os.Getuid() != 0 {
 		t.Skip("requires root")
 	}
-	fd, err := unix.Openat2(unix.AT_FDCWD, ".", &unix.OpenHow{Flags: unix.O_RDONLY})
-	if err != nil {
+	if err := tryOpenat2(t); err != nil {
 		t.Skipf("openat2: %v (old kernel? need Linux 5.6+)", err)
 	}
-	unix.Close(fd)
 }
 
-func TestMountedBy(t *testing.T) {
-	requireOpenat2(t)
+func tryOpenat2(t *testing.T) error {
+	t.Helper()
```

Comment:
AFAIK it does not have to be a helper (so also no need to pass `t`).

Reply:
Yeah, probably not strictly needed; I usually add it by default, so that issues get reported with the line of the Test, not the line in the _helper_. I'll remove